packages/plugin-session-replay-browser/test/session-replay.test.ts

```diff
@@ -247,27 +246,82 @@ describe('SessionReplayPlugin', () => {
   });
 
   describe('recordEvents', () => {
+    test('should send the first two emitted events immediately', () => {
+      const sessionReplay = sessionReplayPlugin();
+      const send = jest.spyOn(sessionReplay, 'send').mockReturnValueOnce(Promise.resolve());
+      sessionReplay.config = mockConfig;
+      sessionReplay.recordEvents();
+      expect(sessionReplay.events).toEqual([]);
```

Comment:
Can we add a comment for what each expect is doing? This seems to be saying "right after setup, there's no session data to send, so we shouldn't have sent any session events" - is that right? How is it different than the next expect on line 258?

Reply:
Yeah I can add some more comments